spec/features/user_can_create_an_account_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "rails_helper"
+
```

Comment:
indentation in this file is not consistent with the rest of the project

Reply:
make sure you indent using spaces. Otherwise, git will not recognize the changes.

Reply:
Confirmed